main.js

```diff
@@ -0,0 +1,47 @@
+function login(){
+    function newLoginHappened(user){
+        if (user){
+            //User is signed in
+            app(user)
+            
+        }else {
```

Comment:
Please follow https://google.github.io/styleguide/jsguide.html in your code formatting.